src/app/features/register/register.component.spec.ts

```diff
@@ -8,7 +9,8 @@ describe('RegisterComponent', () => {
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      declarations: [RegisterComponent]
+      declarations: [RegisterComponent],
+      schemas: [NO_ERRORS_SCHEMA],
```

Comment:
NO_ERRORS_SCHEMA to suppress error message popped out during unit testing since RegisterComponent is using a reusable component (InputFieldComponent) as a child. We are not going to add the child component here as we want to isolate the testing specific to this component only.